### PR TITLE
feat: Add basic routes

### PR DESCRIPTION
**Pull request by davbrito**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const URL_BASE = "https://swapi.dev/api";
+
+export function fetchPeople() {
+  return fetch(`${URL_BASE}/people`);
+}
```
Comment (JuanVqz) [inline src/api.js:5]: :rocket:

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      ul {
+        margin-left: auto;
+        list-style: none;
+        padding: 0;
+        display: flex;
+      }
+    `;
+  }
+
+  #updateOutlet(node) {
```
Comment (JuanVqz) [inline src/components/App.js:46]: why are you adding `#` as a prefix? is it part of Lit? Do you have the documentation link for this?

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          <ul>
+            <li>
+              <a href="/people">People</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main>
+        <!-- Route outlet -->
+        <div ${ref(this.#updateOutlet)}></div>
```
Comment (JuanVqz) [inline src/components/App.js:64]: I can see, I guess it's part of the router, isn't it?

File: src/components/App.js (hunk 2 above)
Comment (davbrito) [reply src/components/App.js:46]: @JuanVqz It's JavaScript native [Private Class Method](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Classes/Private_class_fields) syntax

File: src/components/App.js (hunk 3 above)
Comment (davbrito) [reply src/components/App.js:64]: `ref` is a Lit directive that allows you to retrieve a reference to an element rendered into the DOM. In this case it's the node that I use as the route outlet.

Edit: Here's the documentation for that https://lit.dev/docs/templates/directives/#ref

File: src/components/App.js (hunk 2 above)
Comment (JuanVqz) [reply src/components/App.js:46]: Interesante 🧐

**Review by JuanVqz — APPROVED** [review]
Te volaste la barda!, muy buena aportación.

**Pull request merged**